### DISCUSSION TITLE
Improve Form Block to Accept Frontend Post Submission

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-blocks#1205] [Feature Requests]
The Contact Form block is too basic and easily bettered by other plugins. Let's turn it into a post submission block so people can send in posts

We already have the basic structure for this. It could be a simpler version of a plugin like this: https://wordpress.org/plugins/user-submitted-posts/

**Comment by htmgarcia** [comment]
@stevejburge basically a form submission would create a non-published post ready for review, correct?

**Comment by stevejburge** [comment]
@htmgarcia Yes, that's it
